<launch>
	<arg name="veh"/>
	<arg name="config" default="baseline"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="local" default="false" doc="true to launch locally on laptop. false to launch of vehicle"/>
	<arg name="pkg_name" default="mocap" doc="name of the package"/>
	<arg name="node_name" default="mocap_localization_node" doc="name of the node"/>

	<group ns="$(arg veh)">

		<!--remap from="mocap_localization_node/tag_detections" to="tag_detections"/-->
	    <!-- Local -->
	    <node if="$(arg local)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	        <!---rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/-->
	    </node>
	    <!-- Remote -->
		<include unless="$(arg local)" file="$(find duckietown)/machines"/>
	    <node unless="$(arg local)" machine="$(arg veh)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	        <!--rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/-->
	    </node>
	</group>

</launch>